roles/openshift/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Common variables
+openshift_env: 'test'
+openshift_roles: [ 'master', 'node' ]
+openshift_bind_ip: "{{ ansible_default_ipv4.address }}"
+openshift_public_ip: "{{ ansible_default_ipv4.address }}"
```

Comment:
@twiest what would you think about adding warnings when the defaults are used? or at least the common default values that could cause issues like the public_ip, env, etc?